libs/shared/utils/src/lib/tools/environment-variable-file.ts

```diff
@@ -0,0 +1,48 @@
+import { APIVariableTypeEnum } from 'qovery-typescript-axios'
+import { EnvironmentVariableSecretOrPublic } from '@qovery/shared/interfaces'
+
+// todo remove the any once the api is updated
+export const environmentVariableFile = (variable: EnvironmentVariableSecretOrPublic): boolean => {
+  if (variable.variable_type === APIVariableTypeEnum.ALIAS) {
+    if (
+      variable.aliased_variable?.variable_type === APIVariableTypeEnum.FILE ||
+      variable.aliased_secret?.variable_type === APIVariableTypeEnum.FILE
+    ) {
+      return true
+    }
+  }
+  if (variable.variable_type === APIVariableTypeEnum.OVERRIDE) {
+    if (
+      variable.overridden_variable?.variable_type === APIVariableTypeEnum.FILE ||
+      variable.overridden_secret?.variable_type === APIVariableTypeEnum.FILE
+    ) {
+      return true
+    }
+  }
+  return variable.variable_type === APIVariableTypeEnum.FILE
+}
+
+export const getEnvironmentVariableFileMountPath = (
+  variable: EnvironmentVariableSecretOrPublic | undefined
+): string => {
+  if (!variable) return ''
+  if (variable.variable_type === APIVariableTypeEnum.ALIAS) {
+    if (variable.aliased_variable?.variable_type === APIVariableTypeEnum.FILE) {
+      return variable.aliased_variable.mount_path
+    }
+
+    if (variable.aliased_secret?.variable_type === APIVariableTypeEnum.FILE) {
+      return variable.aliased_secret.mount_path
+    }
```

Comment:
This function is quite difficult to read (lots of if) maybe you can add more spacing or small comments ?